src/EnvironmentCheckSuite.php

```diff
@@ -199,6 +199,12 @@ public static function register($names, $check, $title = null)
         if (!is_array($names)) {
             $names = array($names);
         }
+
+        // Support for omitted namespaces
```

Comment:
Namespace guessing is a bit of a code smell. Would suggest routing through Injector instead so that we can provided named services that have the shortcut names. For comparison, this is how DBField subclasses are looked up from `$db` values.